docs/usage/fields.md

```diff
@@ -0,0 +1,179 @@
+
+The `Field` class is used to define fields in models.
```

Comment:
```suggestion
The `Field` class is used to customize and add metadata to fields of models.
```

Reply:
I'll work on this file. I'll include this. 🙏

Reply:
Not too late I'll add it to my next docs PR. Do you plan to add more suggestions?

Reply:
No all good, I didn't have any more queued up I was just starting to review